### ISSUE TITLE
Clicking on an item in the gallery, sometimes trigger multiple events

### ISSUE THREAD

**Issue by tc-wleite** [sepinf-inc/IPED#1299]
When the user clicks on an item in the gallery, sometimes more than one event to open the file in the viewer is triggered.
It is harmless, but in some cases (specially when the evidence is an UFDR), the following exception is thrown and added to the application log:
```
2022-09-01 08:38:50	[INFO]	[app.ui.FileProcessor]			Opening iphone11.ufdr/(...)/80x80_iOS.png
java.io.IOException: java.lang.InterruptedException
	at iped.engine.io.ZIPInputStreamFactory.getSeekableInputStream(ZIPInputStreamFactory.java:244)
	at iped.engine.io.UFDRInputStreamFactory.getSeekableInputStream(UFDRInputStreamFactory.java:22)
	at iped.engine.data.Item.getSeekableInputStream(Item.java:562)
	at iped.viewers.ImageViewer.loadFile(ImageViewer.java:110)
	at iped.viewers.api.AbstractViewer.loadFile(AbstractViewer.java:112)
	at iped.viewers.MultiViewer.loadFile(MultiViewer.java:108)
	at iped.app.ui.ViewerController.loadInViewer(ViewerController.java:331)
	at iped.app.ui.ViewerController.updateViewer(ViewerController.java:296)
	at iped.app.ui.ViewerController.loadFile(ViewerController.java:259)
	at iped.app.ui.FileProcessor.process(FileProcessor.java:175)
	at iped.app.ui.FileProcessor.doInBackground(FileProcessor.java:128)
	at iped.app.ui.FileProcessor.doInBackground(FileProcessor.java:52)
	at java.desktop/javax.swing.SwingWorker$1.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.desktop/javax.swing.SwingWorker.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.InterruptedException
	at java.base/java.util.concurrent.FutureTask.awaitDone(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.get(Unknown Source)
	at iped.engine.io.ZIPInputStreamFactory.getSeekableInputStream(ZIPInputStreamFactory.java:241)
	... 17 more
2022-09-01 08:38:50	[INFO]	[app.ui.FileProcessor]			Opening iphone11.ufdr/(...)/80x80_iOS.png
```

This log snippet came from a single mouse click, and triggered 2 "Opening <file>" events.
It doesn't happen every time, but it is easy to reproduce, at least using UFDRs in my environment.

**Comment by lfcnassif** [comment]
This is odd, I'll try to reproduce soon.

**Comment by lfcnassif** [comment]
Closed by #1301.
